lib/util/platform.js

```diff
@@ -679,13 +679,17 @@ shaka.util.Platform = class {
         maxResolution.height = 1080;
       }
     } else if (Platform.isWebOS()) {
+      // WebOS has always been able to do 1080p.  Assume a 1080p limit.
+      // Set reasonable defaults in case an issue occurs
+      maxResolution.width = 1920;
+      maxResolution.height = 1080;
```

Comment:
This is not necessary because the catch already has this information in case it fails

Reply:
You would expect the catch should have worked initially then?
I can remove just not sure what issue there is setting defaults initially?

Reply:
We'll leave it at that for now.